.github/workflows/build-operator-images.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and Publish RSCT Operator
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v**'
+
+env:
+  GO_VERSION: "1.22"
+  IMAGE_TAG_BASE: ghcr.io/${{ github.repository }}
+  VERSION: "0.0.1"
+  IMAGE_BUILDER: "docker"
+  
+jobs:
+  build-and-publish-operator:
+    name: Build and Publish RSCT Operator Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: make
```

Comment:
I'm more sure what do you mean by dependencies here? why do we need them? don't we install them while calling them while triggering individual targets like bundle, bundle-build etc..

Reply:
Yes. My bad, will remove this step.